lib/mix/test/mix/tasks/archive_test.exs

```diff
@@ -70,6 +70,23 @@ defmodule Mix.Tasks.ArchiveTest do
     end
   end
 
+  test "archive install invalid file" do
+    in_fixture "invalid_archive", fn ->
```

Comment:
I was concerned building an archive in the same directory as other tests do will cause test pollution but it doesn't seem the case.